### PR TITLE
Add Crates information to the README file

### PR DESCRIPTION
**Pull request by dicortazar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +36,12 @@ $ python3 setup.py install
 
 ## Examples
 
+### Crates
+
+```
+$ perceval crates --category
```
Comment (sduenas) [inline README.md]: You have to set the category. It doesn't work that way.

Comment (dicortazar) [reply README.md]: Should I specify the options here (eg [summary|crates])? Or should I removed that option and just leave 'perceval crates'?

Thanks!

Comment (sduenas) [reply README.md]: Just write an example with any of the categories available

**Comment by dicortazar** [comment]
I think this is already done, please let me know any other change :).

Thanks!

**Pull request closed without merging**